### PR TITLE
sigfigs removed

### PR DESCRIPTION
**Pull request by ruseinov**
Fixes https://github.com/iov-one/weave/issues/207

### CODE REVIEW

File: errors/doc.go [hunk 1]
```diff
@@ -1,19 +1,19 @@
 /**
- Package errors implements custom error interfaces for weave.
+Package errors implements custom error interfaces for weave.
```
Comment (ethanfrey) [inline errors/doc.go]: ???

This is part of the other PR, no?

Comment (ruseinov) [reply errors/doc.go]: well, that's what happens after go fmt. I will remove this change

**Comment by ethanfrey** [comment]
Nice

**Comment by ruseinov** [comment]
@ethanfrey cool, merging that then

**Pull request merged**